src/main/java/org/mockito/internal/progress/MockingProgressImpl.java

```diff
@@ -31,7 +31,7 @@
 
     private OngoingStubbing<?> ongoingStubbing;
     private Localized<VerificationMode> verificationMode;
-    private Location stubbingInProgress = null;
+    private static Location stubbingInProgress = null;
```

Comment:
This won't work in case subbings are made in multiple threads. e.g. when tests are executed parallel

Reply:
True, will check in detail when I find some time.